iOS-Email-Client/Controllers/EmailDetailViewController.swift

```diff
@@ -201,10 +201,8 @@ extension EmailDetailViewController: UITableViewDelegate, UITableViewDataSource{
         cell.delegate = self
         target = cell.moreOptionsContainerView
         if(isGroupable()){
-            DispatchQueue.main.async {
```

Comment:
So Mayer's bug is fixed by removing the `DispatchQueue`. What was the original reason of this code?